web/src/javascript/app.js

```diff
@@ -35,5 +35,19 @@ function updateLeaderBoard() {
   setTimeout(updateLeaderBoard, 1000/leaderboardRefresh - elapsed);
 }
 
+function initButtonOnClick() {
+  const targetLinesBtn = $("#targetLinesBtn")[0];
+  targetLinesBtn.onclick = function() {
+    //const targetLinesBtn = $("#targetLinesBtn")[0];
+    if (targetLinesBtn.className == "btn btn-primary") {
+      document.getElementById("targetLinesBtn").className = "btn btn-default";
+    }
+    else{
+      document.getElementById("targetLinesBtn").className = "btn btn-primary";
+    }
+  };
+}
+
+initButtonOnClick()
```

Comment:
I think it's nice to call it here, but maybe it could be defined in `game.js`? I think it would the right place, since it doesn't really need to be in this file (doesn't require any state from here).